table/get_context.h

```diff
@@ -149,6 +149,8 @@ class GetContext {
 
   bool NeedTimestamp() { return timestamp_ != nullptr; }
 
+  size_t TimestampSize() { return ucmp_->timestamp_size(); }
```

Comment:
If we want to add this as a function, I think it needs to be placed in the `private` section. Otherwise, we can inline it too.